internal/template/utilization_functions.go

```diff
@@ -72,6 +71,39 @@ scalar(
 	return renderUtilization(memoryUtilizationQueryTemplate, data, labelArgs...)
 }
 
+func cpuRequests(data MetricData, labelArgs ...string) (string, error) {
+	cpuResourcesQueryTemplate := `
+scalar(
+  sum(
+    avg_over_time(kube_pod_container_resource_requests_cpu_cores[{{ .Range }}])
+    *
+    on (pod) group_left kube_pod_labels{{ .Labels }}
+  )
+)`
+
+	return renderUtilization(cpuResourcesQueryTemplate, data, labelArgs...)
+}
+
+func memoryRequests(data MetricData, labelArgs ...string) (string, error) {
+	// Returns a value in MB
+	memoryResourcesQueryTemplate := `
+scalar(
+  sum(
+    avg_over_time(kube_pod_container_resource_requests_memory_bytes[{{ .Range }}])
+    *
+    on (pod) group_left kube_pod_labels{{ .Labels }}
+  )
+  /
+  1024
+  /
+  1024
+  /
+  1024
```

Comment:
I think this is the only bit I'm questioning, it seems like it would be easier and safer to just leave this in `B` instead of `GiB` (not `MB` like the comment). I'm pretty sure it would be safe to do something like:
```
query: '{{ memoryRequests . "foo=bar" }} / 1024 / 1024 / 1024'
```